src/System.Threading.Thread/src/System/Threading/Thread.cs

```diff
@@ -209,12 +305,14 @@ public void Resume()
         public static void BeginThreadAffinity() { }
         public static void EndThreadAffinity() { }
 
+#if !NETNATIVE
```

Comment:
Oh right, it wasn't possible to use an alias to choose the one from S.Collections since it is a generic type. Need to figure something out for this.